components/feature/qr/lint.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<lint>
+    <issue id="IconMissingDensityFolder" severity="ignore" />
+    <issue id="Overdraw" severity="ignore" />
```

Comment:
Let's suppress `Overdraw` in the file where it's actually happening. In general we want to avoid it and get notified if we add another one. :)

Reply:
Fixed the overdraw issue.